### PR TITLE
Get enabled and disabled apps from same request in acceptance tests

### PR DESCRIPTION
**Pull request by dpakach**
## Description

In acceptance test use same http request to get enabled and disabled apps.

## Related Issue

- Related https://github.com/owncloud/QA/issues/621

## Motivation and Context

Because of lots of tests the CI has become very slow and takes a lot of time to finish especially in apps like encryption where all core tests are executed multiple times. This improves the code by a little bit so that it can run a bit faster.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]

**Review by paurakhsharma — COMMENTED** [review]
Looks good to me :+1:

File: tests/acceptance/features/bootstrap/Provisioning.php [hunk 1]
```diff
@@ -2775,8 +2775,16 @@ public function cleanupGroups() {
...
-		$this->enabledApps = $this->getEnabledApps();
-		$this->disabledApps = $this->getDisabledApps();
+		SetupHelper::init(
+			$this->getAdminUsername(),
+			$this->getAdminPassword(),
+			$this->getBaseUrl(),
+			$this->getOcPath()
+		);
+		$this->runOcc(['app:list', '--output json']);
+		$apps = \json_decode($this->getStdOutOfOccCommand());
```
Comment (individual-it) [inline tests/acceptance/features/bootstrap/Provisioning.php]:
```suggestion
		$apps = \json_decode($this->getStdOutOfOccCommand(), true);
```
to get an array

**Review by individual-it — CHANGES_REQUESTED** [review]
:+1: maybe just place that all into a new function

**Review by phil-davis — APPROVED** [review]

**Review by individual-it — APPROVED** [review]

**Comment by dpakach** [comment]
Backport on #35372

**Pull request merged**